### ISSUE TITLE
Top 10 Programming Languages to Learn in 2016

### ISSUE THREAD

**Issue by fragmatyc** [ashleygwilliams/breakfast-repo#78]
https://www.youtube.com/watch?v=Z56GLRXxh88

**Comment by texastoland** [comment]
Wow I disagree with 9/10 having written code in 8 of them (24 total). In my mind there are 4 potential reasons to learn a new language:
1. Your boss made you. No argument here.
2. Make more money. Enterprise markets produce the best salary rates. I wish I had **specialized** on libraries and platforms in demand rather playing Groundhog Day with languages. _Angular 2_ and _Salesforce Lightning_ exemplify enterprise JavaScript.
3. Build cool stuff. That'd be a great reason to learn Swift (iOS native) or Clojure (Android native) but JavaScript already enables you to build almost anything (_React Native_).
4. Learn to think. In that case I'd recommend something like Elm (or any Haskell clone) or ClojureScript (or any Lisp) or anything FP (including Rust) or pushing the boundaries on language design (like Emily).

**Comment by fragmatyc** [comment]
I also disagree with the list, but from your bullet points, there's maybe one more. For fun! I learned a tiny little bit of Japanese, Russian, Greek, Portuguese, Latin and Spanish just for fun. This helps develop your brain, your creativity and it changes the way you think. I think it's the same for programming languages. You should first learn for fun, for curiosity, even if you don't use it in your day to day job. It makes better humans :)

**Comment by texastoland** [comment]
Lol I bet @ashleygwilliams would agree too :+1:
